tests/TestCase/Core/Configure/Engine/JsonConfigTest.php

```diff
@@ -96,7 +96,7 @@ public function testReadWithNonExistentFile()
      * Test reading an empty file.
      *
      * @expectedException \Cake\Core\Exception\Exception
-     * @expcetedExceptionMessage Decoding JSON config file "empty.json" did not return any array
+     * @expectedExceptionMessage Decoding JSON config file "empty.json" did not return an array
```

Comment:
This is the test that is failing... To fix it, just copy the actual error message from travis and replace the current text with it.